sunpy/net/vso/attrs.py

```diff
@@ -400,7 +400,12 @@ class Quicklook(_VSOSimpleAttr):
     Quicklook products are *not* searched by default.   Reference:
     documentation in SSWIDL routine vso_search.pro.
     """
-    pass
+    def __init__(self,value):
+        super(Quicklook,self).__init__(value)
+        if(self.value == True):
```

Comment:
In this case arent, value == True and self.value == True the same here, since self.value is assigned at line 404?